backend/kv/sqlite.c

```diff
@@ -275,7 +275,7 @@ backend_fini (void)
 static
 JBackend sqlite_backend = {
 	.type = J_BACKEND_TYPE_KV,
-	.component = J_BACKEND_COMPONENT_SERVER,
+	.component = J_BACKEND_COMPONENT_SERVER | J_BACKEND_COMPONENT_CLIENT,
```

Comment:
What is the reason for this change?

Reply:
Without the backend on the client side, testing would require a running julea-server.
Maybe there could be a compile option which allows any backend to be loaded client side during testing?